src/RedisCache/RedisCache/help/Remove-AzRedisPrivateEndpointConnection.md

```diff
@@ -0,0 +1,183 @@
+﻿
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.RedisCache.dll-Help.xml
+Module Name: Az.RedisCache
+online version: https://docs.microsoft.com/powershell/module/az.rediscache/remove-azredisprivateendpointconnection
+schema: 2.0.0
+---
+
+# Remove-AzRedisPrivateEndpointConnection
+
+## SYNOPSIS
+Remove a Private Endpoint Connection from a Redis Cache.
+
+## SYNTAX
+
+### NormalParameterSet (Default)
+```
+Remove-AzRedisPrivateEndpointConnection [-ResourceGroupName <String>] -Name <String> -PrivateEndpointConnectionName <String> [-Force] [-PassThru]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Remove a private endpoint connection from a Redis Cache.
+
+## EXAMPLES
+
+### Example 1: Remove a private endpoint connection and return the result
+```
+PS C:\>Remove-AzRedisPrivateEndpointConnection -Name "mycache" -PrivateEndpointConnectionName "MyPrivateEndpoint.abcd123e45" -Force -PassThru
+True
+```
+
+This command removes a private endpoint connection named **MyPrivateEndpoint.abcd123e45** from Redis Cache named **mycache** and displays whether the operation is successful. 
+
+### Example 2: Remove a private endpoint connection and do not display the result
+```
+PS C:\>Remove-AzRedisPrivateEndpointConnection -Name "mycache" -PrivateEndpointConnectionName "MyPrivateEndpoint.abcd123e45" -Force
+```
+
+This command removes a private endpoint connection named **MyPrivateEndpoint.abcd123e45** from Redis Cache named **mycache**. Since the *PassThru* parameter is not specified, the result of the operation is not displayed.
```

Comment:
```suggestion
This cmdlet removes a private endpoint connection named *MyPrivateEndpoint.abcd123e45* from Azure Cache for Redis named *mycache*. Because the **PassThru** parameter is not specified, the result of the operation is not displayed.
```